schemas/data_model.py

```diff
@@ -0,0 +1,204 @@
+import core.const as enums
+
+SCHEMA = {
+    "Programme Progress": {
+        "columns": {
+            "Submission ID": "str",
+            "Programme ID": "str",
+            "Question": "str",
+            "Answer": "str",
+        },
+        "uniques": ["Submission ID"],
+        "foreign_keys": {"Programme ID": {"parent_table": "Programme_Ref", "parent_pk": "Programme ID"}},
+    },
+    "Programme_Ref": {
+        "columns": {
+            "Programme ID": "str",
+            "Programme Name": "str",
+            "FundType_ID": "str",
+            "Organisation ID": "str",
+        },
+        "uniques": ["Programme ID"],  # TODO: Assuming an Organisation can have multiple programmes
+        "foreign_keys": {"Organisation ID": {"parent_table": "Organisation_Ref", "parent_pk": "Organisation ID"}},
+    },
+    "Organisation_Ref": {
+        "columns": {
+            "Organisation ID": "str",
+            "Organisation": "str",
+            "Geography": "str",
+        },
+        "uniques": ["Organisation ID", "Organisation"],  # TODO: Assuming we dont want multiple Orgs with the same name
+    },
+    "Project Details": {
+        "columns": {
+            "Submission ID": "str",
+            "Project ID": "str",
+            "Primary Intervention Theme": "str",
+            "Single or Multiple Locations": "str",
+            "Locations": "str",
+        },
+        "uniques": ["Submission ID", "Project ID"],  # TODO: Is there only ever one project per submission?
```

Comment:
so there can be multiple projects per submission. But these could be unique-together. 

Does the format of your "uniques" value (in a list) mean unique together (like an index), or that each is unique individually?

Reply:
I'm not quite sure what I meant by that comment... should either submission or project IDs be unique on their own?

Reply:
NOTE TO SELF: Change this to Project ID and Project Name and add Project Name

Reply:
I've changed this to just `"uniques": ["Project ID"]` now, because you mentioned we can map Project ID to Council and Project Name now? Correct me if I'm wrong about this... @nsnape